docs/platform/deploying-airbyte/chart-v2-community.mdx

```diff
@@ -0,0 +1,338 @@
+
+:::important Enterprise customers
+Follow the [Self-Managed Enterprise guide](../enterprise-setup/chart-v2-enterprise) instead.
+:::
+
+## What's a Helm chart?
+
+Airbyte runs on [Kubernetes](https://kubernetes.io/). [Helm](https://helm.sh/) is a package manager for Kubernetes. A [Helm chart](https://helm.sh/docs/topics/charts/) is a collection of files that describe a related set of Kubernetes resources. Think of Helm charts as the method Airbyte uses to define, install, and manage its Kubernetes application. When you install or upgrade Airbyte, you pull in a specific Helm chart representing that version.
+
+## Why you should upgrade
+
+Upgrading to the new Helm chart now has the following benefits.
+
+1. By upgrading in advance, you can schedule this upgrade for a convenient time. Avoid blocking yourself from upgrading Airbyte to a future version when the new chart is mandatory and you're busy.
+
+2. The new Helm chart is more aligned with Helm's best practices for chart design.
+
+3. The new Helm chart has broader and more detailed options to customize your deployment. In most cases, it's no longer necessary to specify environment variables in your `values.yaml` file because the chart offers a more detailed interface for customization. If you do need to use environment variables, you can use fewer of them.
+
+## Which versions can upgrade to Helm chart V2
+
+The following versions of Airbyte can use Helm chart V2:
+
+- Airbyte version 1.6.0 and later, if installed and managed with Helm _and_ connected to an external database, storage, and secrets manager
+
+The following versions of Airbyte _can't_ use Helm chart V2:
+
+- Airbyte versions before 1.6.0
+
+- Airbyte versions installed and managed with abctl
+
+- Airbyte versions that aren't connected to an external database, storage, and secrets manager
+
+:::important
+Airbyte is evaluating possible migration paths to Helm chart V2 for those that don't meet the criteria above.
+:::
+
+## How to upgrade
+
+In most cases, upgrading is straightforward. To upgrade to Helm chart V2, you complete the following steps.
+
+1. Ensure you have configured Airbyte to use an external database, external bucket storage, and external secrets manager.
+
+2. Prepare to deploy a fresh installation of Airbyte.
+
+3. Create a new `values.yaml` file.
+
+4. Deploy a new version of Airbyte using your new `values.yaml` file and the new Helm chart version.
+
+### Configure external database, bucket storage, and secrets manager
+
+<!-- This might be a non-starter for community. Can we reasonably ask them to do this? -->
+
+<!-- Although Airbyte provides its own basic database, storage, and secrets management, most people opt to use their own. If you don't already do this, you must enable these before migrating.
```

Comment:
Would it help to add some explanation for why we want to recommend setting up an external database, storage, and secrets manager?

The V2 chart does still support the basic embedded versions of these services like the V1 chart did, so I could potentially see how some users would not understand why this is listed as a step for upgrading the chart when it technically isn't required.